### ISSUE TITLE
Support to run entirely locally with LocalAI

### ISSUE THREAD

**Issue by mudler** [stitionai/devika#204]
First of all - thank you for your hard work, from a maintainer to a maintainer, I think projects like this should exist - so I'm happy to see the community to step up!

I'd like to use devika, but I want to use it 100% Local, for instance with LocalAI ( https://github.com/mudler/LocalAI ) which is an OpenAI free alternative.

Some context:

I'm the author of LocalAI, I can likely help in anything you need from the backend to make this working properly.

**Comment by shrinivaspatwardhan** [comment]
That's interesting, I am willing to contribute as well.

**Comment by mybaseai** [comment]
@mudler great suggestion!

**Comment by obliviousz** [comment]
Hi I am new to this(open source on AI), but isn't using Ollama for llama2 an offline/local procedure? 

Thanks!

**Comment by shrinivaspatwardhan** [comment]
that's correct. infact a similar project is OpenDevin  that's a open-source version of Devin that i came across today

**Comment by franklinselva** [comment]
@shrinivaspatwardhan Would you be willing to take it as a integration of LocalAI into Devika?
